backend/src/main/resources/db/migration/V23__Drop_unique_team_request_constraint.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE teamRequest DROP CONSTRAINT unique_team_user_request, ADD CONSTRAINT unique_team_user_request UNIQUE (team_id, user_id, requester_id);
```

Comment:
Shouldn't it just be team and user that way a user can't request to join a team that they already were requested to be on

Reply:
No, user_id is the person being requested to join that team or requesting to join the team user_id will never be the creator of the team